dgraphtest/paths.go

```diff
@@ -56,4 +56,10 @@ func init() {
 	binDir = filepath.Join(basePath, "binaries")
 	encKeyPath = filepath.Join(basePath, "data", "enc-key")
 	aclSecretPath = filepath.Join(basePath, "data", "hmac-secret")
+
+	log.Printf("[INFO] baseRepoDir: %v", baseRepoDir)
```

Comment:
can you add a comment to as to why?

Reply:
These are just debugging statements which could be useful when tests fail. Do you have a suggestion as to what I could write here?